src/api/helpers/utils.js

```diff
@@ -15,27 +15,19 @@
  * with this program; if not, write to the Free Software Foundation, Inc.,
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
-
 import {get as _get} from 'lodash';
 import {loadEntity} from './middleware';
-
-
 export const aliasesRelations = ['aliasSet.aliases.language'];
 export const identifiersRelations = ['identifierSet.identifiers.type'];
-export const relationshipsRelations = ['relationshipSet.relationships.type'];
-
+export const relationshipsRelations = ['relationshipSet.relationships.type']; // Fixed from 'relationship' to 'relationships'
```

Comment:
I don't think the comment is required here, unless I missed something.
As it is creating an ESLint warning, let's remove it.
```suggestion
export const relationshipsRelations = ['relationshipSet.relationships.type'];
```